tools/docker/install/install_rocksdb.sh

```diff
@@ -17,20 +17,41 @@
 # Select RocksDB version.
```

Comment:
Hi @bashimao , If possible, please help review this file, your RocksDB PR will be affected by this docker images which is created by the docker file: 
`tfra/nosla-cuda11.2-cudnn8.1-ubuntu18.04-manylinux2010-multipython:latest` 
Thank you ~

Reply:
@rhdong Checked it. Looks good to me.